### PR TITLE
Resize landing page logos for better display 

### PR DESCRIPTION
**Pull request by haticekaratay**
### Description

This pull request is to adjust the size of the logos on the landing page to improve appearance and ensure better visual alignment. Previously they were too large.
Before: 
image
After: The rampviz logo in the video is a placeholder to demonstrate better for the outcome.

https://github.com/user-attachments/assets/408277b6-18bc-4b42-a890-f14941834221

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Review by pllim — APPROVED** [review]
LGTM. Thanks!

**Comment by kecnry** [comment]
Do we understand how this "broke" before (was it an update to RTD styles or something)?  Are all of the CSS changes here necessary?  I'm just a bit worried about unintended consequences to changing global styling - but nothing stands out as being a problem in the [rendered docs](https://jdaviz--3243.org.readthedocs.build/en/3243/), so maybe its ok?

**Comment by haticekaratay** [comment]
> Do we understand how this "broke" before (was it an update to RTD styles or something)? Are all of the CSS changes here necessary? I'm just a bit worried about unintended consequences to changing global styling - but nothing stands out as being a problem in the [rendered docs](https://jdaviz--3243.org.readthedocs.build/en/3243/), so maybe its ok?

I checked all the other pages locally and found that it didn't affect them. Only CSS changes were made to scale the logo by 70% and to add a margin between the button and the logo.

File: docs/_static/jdaviz.css [hunk 1]
```diff
@@ -15,12 +16,6 @@
   margin: 0px;
 }
 
-.sd-card .sd-card-img-top {
-  height: 52px;
-  width: 52px;
-  margin-left: auto;
-  margin-right: auto;
-}
```
Comment (kecnry) [inline docs/_static/jdaviz.css:23]: looks like `:img-top` is no longer used anywhere ✅

**Review by kecnry — APPROVED** [review]
Thanks!

**Comment by pllim** [comment]
> Do we understand how this "broke" before

Hmm yeah, they looked completely different and even compatible with dark mode until v4.0 when rampviz was also added.

https://jdaviz.readthedocs.io/en/v3.9.1/

Screenshot 2024-10-22 133210

**Comment by pllim** [comment]
pydata-sphinx-theme jump was only from 0.15.2 to 0.15.4 between those 2 releases.

File: docs/_static/jdaviz.css (hunk 1 above)
Comment (haticekaratay) [reply docs/_static/jdaviz.css:23]: Yes

**Comment by kecnry** [comment]
hmmm did something else change in the style or index page that broke the layout and background color on the image tiles?

**Comment by pllim** [comment]
Hmm I don't see any CSS changes in https://github.com/spacetelescope/jdaviz/compare/v3.9.1...v4.0.0

**Comment by kecnry** [comment]
Can we downgrade pydata-sphinx-theme to confirm that was the cause?  We can also consider the remaining dark mode changes as a follow-up ticket since they are not introduced by this PR, and this PR _greatly_ improve the layout!

**Comment by pllim** [comment]
Downgrade (#3244) didn't help.

https://jdaviz--3244.org.readthedocs.build/en/3244/

**Comment by kecnry** [comment]
Thanks for checking!  I'd vote for merging this as-is and creating a follow-up ticket to track down and fix the dark support then.

**Comment by pllim** [comment]
Wait

**Review by pllim — CHANGES_REQUESTED** [review]
I have alt fix

**Comment by pllim** [comment]
I was comparing with what astropy did and might have alternate fix. Please hold.

**Comment by haticekaratay** [comment]
Wow, there's been a lot happening since I last checked the chat. Maybe downgrading the sphinx-design or pydata-sphinx-theme could be the solution, as I didn't notice any CSS changes in the file itself. Instead of pinning the dependencies, should we focus on making our docs compatible with the latest versions? I'll test it out and report back here.

**Comment by pllim** [comment]
What about https://jdaviz--3245.org.readthedocs.build/en/3245/ over at #3245 ?

Sorry, I should have opened a PR here back when astropy was patched. The discussions here reminded me of that patch, so thanks!

**Comment by pllim** [comment]
> worth putting on top

No, sorry. This PR is unnecessary with the other PR changes.

**Pull request closed without merging**